velox/docs/functions/spark/json.rst

```diff
@@ -0,0 +1,13 @@
+==============
+Json Functions
+==============
+
+.. spark:function:: json_object_keys(jsonString) -> array(string)
+
+    Returns all the keys of the outermost JSON object as an array if a valid JSON object is given. If it is any other valid JSON string, an invalid JSON string or an empty string or is not a JSON object string, the function returns null. ::
```

Comment:
nit: if a valid JSON object is given -> if a valid JSON object string is given

> If it is any other valid JSON string, an invalid JSON string or an empty string or is not a JSON object string

There are four cases to return NULL as described in the documentation, but in the code there are only three. Would you make them consistent?

Reply:
After looking at the spark doc, there are only three cases to return NULL, I will update the doc